2.1.0 Changes

cs-abhishek-shukla opened the pull request:
- Redrafted release notes
- Included contents.md modifications
- WIP

cs-abhishek-shukla committed:
2.0.0 changes (#18)

* 2.0.0 changes
- release notes
- usage doc
- setup doc
- contents doc

cs-abhishek-shukla committed:
2.0.0 changes (#19)

- upgrade instructions
- Readme link to upgrade instructions
- upgrade instructions

cs-suyog-jinturkar committed:
Update upgrade-instructions.md

cs-abhishek-shukla committed:
Merge pull request #20 from fortinet-fortisoar/cs-suyog-jinturkar-patch-1

Update upgrade-instructions.md

cs-abhishek-shukla committed:
Merge branch 'fortinet-fortisoar:develop' into release/2.1.0

cs-suyog-jinturkar commented on release_notes.md:
```diff
@@ -1,118 +1,57 @@
+Experience a significant boost in performance during the ingestion of Threat Intel Feeds for Outbreak Alerts:
 
-When you click Outbreak Management in the navigation menu, you’re greeted with an intuitive layout featuring these powerful tabs:
+**Bulk Ingestion**: The individual IOC creation process in playbooks has been replaced with a Bulk Ingestion step. This enhancement leverages a connector action to trigger a bulk ingestion playbook that batches IOCs into groups of 1,000 and processes them asynchronously. This approach ensures faster and more efficient processing of threat intelligence feeds.
```
x no. details are not needed here in release notes

cs-suyog-jinturkar commented on release_notes.md:
```diff
@@ -1,118 +1,57 @@
+### Automated Updates for Outbreak Alerts
 
-- **Outbreaks**: Access comprehensive records of all outbreak alerts at your fingertips.
+A new playbook, **Update Outbreak Alert Details**, has been introduced to automatically update critical information, including CVEs, background details, and descriptions for Outbreak Alerts with the statuses *`New`* or *`Tracking`*. This playbook is configured to run daily, ensuring that alerts remain current and accurate.
```
treat this as a feature achieved using pb and not a vice versa

cs-suyog-jinturkar commented on release_notes.md:
```diff
@@ -1,118 +1,57 @@
+### STIX Fields Mapping in Threat Intel Feeds
 
-#### Improved Solution Pack Configuration Experience
+Additional STIX fields have been mapped to Threat Intel Feeds ingested from Outbreak Alerts. These fields include:  
```
adding details of "why"? is important here as well

cs-adarsh-gupta commented on docs/contents.md:
```diff
@@ -7,22 +7,23 @@ The **Outbreak Response Framework** solution pack contains the following resourc
+| `outbreak-auto-install-time-frame`  | Saves the number of days for which the outbreak-specific response solution packs are set to install during configuration.                                                                             |
+| `outbreak-threat-hunt-tools-params` | Saves the outbreak hunt tools parameters, like ADOM name for FortiAnalyzer and `splunk_index` for Splunk, and helps the outbreak configuration wizard to populate these parameters.                   |
+| `outbreak-alert-config`             | Saves the configuration parameters provided at the time of framework configuration, like `huntTools`, `installOutbreakType`, `emailAddress`, etc, helping in outbreak installation and investigation. |
 
```
One more Key Store is there outbreak-threat-hunting-workflow-config

cs-abhishek-shukla replied:
Adding. It was missing from doc input

cs-rahul-vishwakarma commented on release_notes.md:
```diff
@@ -1,118 +1,57 @@
+A new feature automatically updates critical information for Outbreak Alerts with the statuses *New* or *Tracking*. This is achieved using the **Update Outbreak Alert Details** playbook, which updates key details such as CVEs, background information, and descriptions. The playbook is scheduled to run daily, ensuring that alerts remain current and accurate.
 
-- **Hunt Rules**: Explore and manage Fortinet Fabric Rules, Sigma Rules, and YARA Rules linked to your outbreak-specific response solution packs.
+### STIX Fields Mapping in Threat Intel Feeds
```
Sir, replace the entire "### STIX Fields Mapping in Threat Intel Feeds" section with following:

### Fields Mapping in Threat Intel Feeds

- Additional STIX fields have been mapped to Threat Intel Feeds ingested from Outbreak Alerts. These fields include:  
  - `Pattern`
  - `Pattern Version`
  - `Pattern Type`
  These fields offer detailed descriptions of indicators and by mapping them with additional STIX fields, the Threat Intel Feeds align with global standards, making them easier to share and integrate with other tools and platforms. This integration and sharing helps security systems effectively identify, classify, and respond to threats based on defined patterns and indicator types.

- Updated the mapping of the *Type* field in Threat Intel Feeds to accurately reflect the specific hash type (e.g., *FileHash-MD5*, *FileHash-SHA256*, etc.) instead of using the generic type *FileHash*.

cs-suyog-jinturkar commented on docs/contents.md:
Duplicate entry, please check above row and remove this

cs-abhishek-shukla replied:
One is parent, another is child.

cs-suyog-jinturkar commented on release_notes.md:
```diff
@@ -1,118 +1,57 @@
+### Automated Updates for Outbreak Alerts
 
-- **Outbreaks**: Access comprehensive records of all outbreak alerts at your fingertips.
+A new feature automatically updates critical information for Outbreak Alerts with the statuses *New* or *Tracking*. This is achieved using the **Update Outbreak Alert Details** playbook, which updates key details such as CVEs, background information, and descriptions. The playbook is scheduled to run daily, ensuring that alerts remain current and accurate.
```
This needs to be added in usage.md as well as a new section with a screenshot of the schedule and outbreak alert details page before and after of this playbook

cs-suyog-jinturkar replied:
please also mention that user can disable this feature by deactivating the said schedule

cs-abhishek-shukla replied:
Added a note in usage.

cs-suyog-jinturkar commented on release_notes.md:
```diff
@@ -1,118 +1,57 @@
+  - `Pattern Type`
 
-The wizard has received several key upgrades, making your configuration process seamless:
+  These fields offer detailed descriptions of indicators and by mapping them with additional STIX fields, the Threat Intel Feeds align with global standards, making them easier to share and integrate with other tools and platforms. This integration and sharing helps security systems effectively identify, classify, and respond to threats based on defined patterns and indicator types.
```
please note for this here on in usage.md that user will need latest TIM feature

cs-abhishek-shukla replied:
Adding a note in usage...

cs-abhishek-shukla replied:
We need to move this point under TIM because unless user updates TIM, ORF will not reflect these changes; and TIM update is *after* ORF

cs-suyog-jinturkar commented on release_notes.md:
```diff
@@ -1,118 +1,57 @@
+### Enhanced CVE Details with EPSS Scores
 
-#### Improved Investigation Flow
+The following fields have been mapped into CVE records to provide enhanced threat context:  
```
This also be part of usage of md with screenshot showing how it looks like

cs-abhishek-shukla replied:
Waiting for configuration.

cs-abhishek-shukla replied:
Done

cs-suyog-jinturkar commented on release_notes.md:
Please reword this point as Optimised **Investigate Outbreak** playbook by removing duplicate steps. No need to mention about which steps are removed as user is not interested in steps but the functioanality.

cs-abhishek-shukla replied:
Done

cs-suyog-jinturkar commented on release_notes.md:
IMO, no need to mention this point as it will not impact any functionality.

cs-abhishek-shukla replied:
It is a change, nonetheless. Let's keep it.

cs-suyog-jinturkar commented on release_notes.md:
```diff
@@ -1,118 +1,57 @@
+A new feature automatically updates critical information for Outbreak Alerts with the statuses *New* or *Tracking*. This is achieved using the **Update Outbreak Alert Details** playbook, which updates key details such as CVEs, background information, and descriptions. The playbook is scheduled to run daily, ensuring that alerts remain current and accurate.
 
-- **Hunt Rules**: Explore and manage Fortinet Fabric Rules, Sigma Rules, and YARA Rules linked to your outbreak-specific response solution packs.
+### Fields Mapping in Threat Intel Feeds
```
We need to add one section here as well as add more information in the usage.md
UI enhancements (please use better name here)
1. Updated Navigation Manu
a. Brand new logo for Outbreak Management
b. Dedicated navigation menu item for the Outbreak Management dashboard

2. Outbreak Alert detailed view enhancements 
a. Hunts Rules are not part of the individual Outbreak Alerts detailed view 
b. Removed dashboard and hunt rules tabs from the outbreak management 

3. Improved Outbreak Response Overview dashboard by adding one more metrices in it as below;
"Outbreak By Severity (last 30 days)

cs-suyog-jinturkar commented on release_notes.md:
Please add this "Outbreak Config wizard modifications" here as well as in wizard release notes;

**Configure Integrations** Page Enhancements:
    - NIST National Vulnerability Database connector configuration is optional now.
    - Wizard now also verifies whether the **Mark as Default Configuration** option has been selected for all chosen integrations.

cs-adarsh-gupta commented on docs/contents.md:
Two new Playbooks are added with this
1. Link Related Threat Intel Feeds - This playbook links the unlinked Threat Intel Feeds to the Outbreak Alert
2. Find Threat Intel Feeds Records Linked to Outbreak Alert - Finds every Threat Intel Feeds records linked to Outbreak Alert

cs-adarsh-gupta commented on release_notes.md:
Please add one more release note
Earlier only max 5000 IOC records were fetched and investigated, Now All the IOCs linked with Outbreak Alert will be fetched and investigated

cs-abhishek-shukla committed:
2.1.0 changes
- Redrafted release notes
- Included contents.md and usage modifications
- Feedback comments addressed
